_episodes/03-create.md

```diff
@@ -553,35 +553,15 @@ rm: cannot remove `thesis': Is a directory
 This happens because `rm` by default only works on files, not directories.
 
 `rm` can remove a directory *and all its contents* if we use the 
-recursive flag `-r`, however ...
-
-> ## With Great Power Comes Great Responsibility
->
-> Removing the files in a directory recursively can be a very dangerous
-> operation. If we're concerned about what we might be deleting we can
-> add the "interactive" flag `-i` to `rm` which will ask us for confirmation
-> before each step
->
-> ~~~
-> $ rm -r -i thesis
-> rm: descend into directory 'thesis'? y
-> rm: remove regular empty file 'thesis/quotations.txt'? y
-> rm: remove directory 'thesis'? y
-> ~~~
-> {: .language-bash}
->
-> This removes any files in the directory, then the directory itself, asking
-> at each step for you to confirm the deletion.
-{: .callout}
-
-If you haven't already deleted the `thesis` directory using the interactive flag,
-you can delete the directory without any confirmation prompts using the following command:
+recursive flag `-r`, and it will do so *without any confirmation prompts*:
 
 ~~~
 $ rm -r thesis
 ~~~
 {: .language-bash}
 
+Given that there is no way to retrieve files deleted using the shell,
+`rm -r` *should be used with great caution* (you might consider adding the interactive flag `rm -ri`).
```

Comment:
Broken formatting around "caution"?

Reply:
Also, we've squashed the options together here, I think we should keep them separate.

Reply:
I can remove the italic if that's what you're suggesting?

Reply:
Why should the flags be kept separate? Combining them is introduced in episode 2.

Reply:
Ugh, I scanned the line and missed the start of the "*". Sorry.

I'm aware we introduced combining arguments, but I think we should stick to expressing them as individual ones for the lessons for clarity (this brings up whether there's other places where we are combining them...)